dom-querying/main.js

```diff
@@ -0,0 +1,24 @@
+console.log('hello, world');
+
+var a = document.querySelector('h1');
```

Comment:
I would avoid using single letter variable like this in the future! something that has a semantic meaning to it for the data contained is always a good idea!